src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -137,30 +136,50 @@ public void testPeriodic() {
 	}
 
 	public void autonomousPeriodic() {
+		
+		drive.logClosedLoopErrors();
+		drive.publishRawSensorValues();
+//		drive.PositionModeMove(drive.feetToTicks(amountToGoLeft), drive.feetToTicks(amountToGoRight));
+		drive.motionMagicMove(amountToGoLeft, amountToGoRight);
```

Comment:
Comment this out too. When we eventually start working on Autonomous, we won't be using this in here.